udata/core/user/commands.py

```diff
@@ -46,6 +47,20 @@ def create():
 
 
 @m.command
+def activate():
+    '''Activate an existing user (validate their email confirmation)'''
+    email = prompt('Email')
```

Comment:
Interesting that for `delete` we prompt the email and for `set_admin` and `password` we have it as a parameter. Maybe an opportunity to be consistent here? @noirbizarre ?